uvicorn/main.py

```diff
@@ -169,6 +169,14 @@ def print_version(ctx: click.Context, param: click.Parameter, value: bool) -> No
     help="WebSocket per-message-deflate compression",
     show_default=True,
 )
+@click.option(
+    "--ws-ignore-upgrade",
+    type=bool,
+    default=False,
+    help="Upgrade requests will be ignored and answered with normal HTTP responses",
```

Comment:
```suggestion
    help="Upgrade requests will be ignored, and HTTP/1.1 will continue to be used.",
```

Reply:
I think renaming would be in order, I had the same thought before. Changing the help as well.

Reply:
Done